compiler/src/compile.rs

```diff
@@ -2377,16 +2372,24 @@ fn compile_conversion_flag(conversion_flag: ast::ConversionFlag) -> bytecode::Co
 
 #[cfg(test)]
 mod tests {
-    use super::Compiler;
+    use super::{CompileOpts, Compiler, OutputStream};
     use crate::symboltable::make_symbol_table;
     use rustpython_bytecode::bytecode::Constant::*;
     use rustpython_bytecode::bytecode::Instruction::*;
     use rustpython_bytecode::bytecode::{CodeObject, Label};
     use rustpython_parser::parser;
 
+    impl<O> Default for Compiler<O>
+    where
+        O: OutputStream,
```

Comment:
Maybe just have this be a function, rather than only impl-ing Default when a cfg matches